mkconcore.py

```diff
@@ -46,6 +46,7 @@
 prefixedgenode = ""
 sourcedir = sys.argv[2]
 outdir = sys.argv[3]
+

```

Comment:
Can you eliminate this file from the PR? I think you entered a new line here, so it is included.

The reasoning is this file is one of the most used files and often with local changes. Any change made to this file is trouble. :)

Reply:
> > Can you eliminate this file from the PR? I think you entered a new line here, so it is included.
> > The reasoning is this file is one of the most used files and often with local changes. Any change made to this file is trouble. :)
> 
> Sure I will do that right now.

@pradeeban mkconcore.py is now same as original one.
 And also,
Huge Congratulations!!!!! on getting this organisation selected for GSoC 2023.

Reply:
> > > Can you eliminate this file from the PR? I think you entered a new line here, so it is included.
> > > The reasoning is this file is one of the most used files and often with local changes. Any change made to this file is trouble. :)
> > 
> > 
> > Sure I will do that right now.
> 
> @pradeeban mkconcore.py is now same as original one. And also, Huge Congratulations!!!!! on getting this organisation selected for GSoC 2023.

As I Know the selected organizations are not yet announced? Yeah but you are right this will be one of that

Reply:
> > > > Can you eliminate this file from the PR? I think you entered a new line here, so it is included.
> > > > The reasoning is this file is one of the most used files and often with local changes. Any change made to this file is trouble. :)
> > > 
> > > 
> > > Sure I will do that right now.
> > 
> > 
> > @pradeeban mkconcore.py is now same as original one. And also, Huge Congratulations!!!!! on getting this organisation selected for GSoC 2023.
> 
> As I Know the selected organizations are not yet announced? Yeah but you are right this will be one of that

https://summerofcode.withgoogle.com/programs/2023/organizations check!!